test/activities/content/data/EditableContent.js

```diff
@@ -0,0 +1,10 @@
+export const editableContent = {
+	'class': [
+		'content'
+	],
+	'properties': {
+		'name': 'Extra Special Content'
```

Comment:
@CodeBaboon @ChrisKraljevicD2L is this the expected structure for the `name` property on the entity?

Reply:
I think it is `title` but I will try to confirm.

Reply:
This is the activity-usage response for an html content topic on my local instance, appears to confirm `title` as the property name (assuming I'm doing this right):

![image](https://user-images.githubusercontent.com/2243995/92497830-73149e00-f1bf-11ea-9407-1de7ed46328b.png)

Reply:
Awesome, thanks for looking into that. Will adjust accordingly 👍